README.md

```diff
@@ -38,5 +38,81 @@ rails generate blacklight:install
 * Bundler
 * Rails 5.0+
 
-## Configuring Apache Solr 
+## Configuring Apache Solr
 You'll also want some information about how Blacklight expects [Apache Solr](http://lucene.apache.org/solr ) to run, which you can find in [README_SOLR](https://github.com/projectblacklight/blacklight/wiki/README_SOLR)
+
+## Building the javascript
+The javascript is built by npm from sources in `app/javascript` into a bundle
+in `app/assets/javascripts/blacklight/blacklight.js`. This file should not be edited
+by hand as any changes would be overwritten.
+
+This is accomplished with the following steps:
+1. Install npm
+1. run `npm install` to download dependencies
+1. run `npm run js-compile-bundle` to build the bundle
```

Comment:
Running this command generates a JS file with double quotes and other Hound complaints. Should we have Hound ignore this file?